LayoutTests/imported/w3c/web-platform-tests/shadow-dom/reference-target/tentative/property-reflection-expected.txt

```diff
@@ -0,0 +1,4156 @@
+
+PASS button.ariaControlsElements has reflection behavior ReflectsHostInArray when pointing to button with reference targetappendTestDeclaratively
```

Comment:
From the build failures it looks like this file needs updating.

Reply:
Done.